### PR TITLE
Jeremy lig 1666 add docs how to block ip addresses pip

### PR DESCRIPTION
**Pull request by japrescott**
closes lig-1666
- add docs on how one can create a S3 policy with advanced restriction (e.g IP)

[Minimum AWS Policy requirements — lightly 1.2.26 documentation.pdf](https://github.com/lightly-ai/lightly/files/9394859/Minimum.AWS.Policy.requirements.lightly.1.2.26.documentation.pdf)

### CODE REVIEW

File: docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket_minimum_policy.rst [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+Minimum AWS Policy requirements
+===============================
+
+It is possibly to make your access policy very restrictive and to even deny anyone with the correct IAM user credentials or role from outside of e.g your VPC or of a certain IP range from reading your data.
+
+The only hard requirement Lightly requires to properly work is `S3:ListBucket`.
+With this permission Lightly will only be able to list the filenames within your bucket but can't actually access the contents of your data. Only you will be able to access your data's content.
+
+.. warning::
+    The Lightly Worker will need to be running within the permissioned zone (e.g within your VPC or IP range) and will need the configuration flag `datasource.bypass_verify` set to `True`.
```
Comment (guarin) [inline docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket_minimum_policy.rst:12]: Maybe mention when and where exactly `datasource.bypass_verify` has to be set. Should it be passed as argument when starting the worker or can it also be passed as part of the `worker_config` when scheduling a run?

**Review by guarin — APPROVED** [review]
Awesome! Just one small suggestion :)

File: docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket_minimum_policy.rst (hunk 1 above)
Comment (japrescott) [reply docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket_minimum_policy.rst:12]: @guarin I tested it with passing it as config to a scheduled job. I am not sure if both work out of the box? can you enlighten me? It is added to the `docker.yml`.

Comment (guarin) [reply docs/source/getting_started/dataset_creation/dataset_creation_aws_bucket_minimum_policy.rst:12]: Looking at the code the `worker_config` overwrites the argument. So it should be set in `worker_config`.

**Pull request merged**